packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx

```diff
@@ -81,21 +83,46 @@ export const DeckLayout = ({
     }
   }, [layoutParts]);
 
-  const searchEnabled = !!usePlugin('dxos.org/plugin/search');
-  const activeId = useMemo(() => Array.from(attention.attended ?? [])[0], [attention.attended]);
+  const [activeId, setActiveId] = useState<string | undefined>(Array.from(attention.attended ?? [])[0]);
+  const deckRef = useRef<HTMLDivElement>(null);
+  useLayoutEffect(() => {
+    let t: NodeJS.Timeout | undefined;
```

Comment:
If timeouts are just numbers, maybe just `number` since this is a browser runtime? Unclear by its name whether it’s portable.